test/contentsharecontroller/ContentShareMediaStreamBroker.test.ts

```diff
@@ -0,0 +1,198 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import * as chai from 'chai';
+import * as sinon from 'sinon';
+import { SinonStub } from 'sinon';
```

Comment:
We do not recommend using the Sinon stub. Since you used it only once, please replace it with a test class extending DefaultDeviceController.

https://github.com/aws/amazon-chime-sdk-js/blob/f8c7aa9bc1c6d42d510f72ca4f74bfb0f7aacccd/script/check-code-style.js#L113